tests/test_bake_project.py

```diff
@@ -269,7 +269,7 @@ def test_bake_with_no_console_script(cookies):
 
 
 def test_bake_with_console_script_files(cookies):
-    context = {'command_line_interface': 'click'}
+    context = {'command_line_interface': 'Click'}
     result = cookies.bake(extra_context=context)
```

Comment:
**Suggestion:** The 'command_line_interface' value in the context dictionary is case sensitive. It's better to use lower case to avoid potential issues with case sensitivity in the future.
```suggestion
    context = {'command_line_interface': 'click'}
    result = cookies.bake(extra_context=context)
```